p2p/messageCheck/messageVerifier_test.go

```diff
@@ -0,0 +1,123 @@
+package messagecheck_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/p2p/message"
+	messagecheck "github.com/ElrondNetwork/elrond-go/p2p/messageCheck"
+	"github.com/ElrondNetwork/elrond-go/p2p/mock"
+	"github.com/ElrondNetwork/elrond-go/testscommon"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewMessageVerifier(t *testing.T) {
```

Comment:
beside this, we might create an integration test in which we have 2 connected peers that and verify their received messages with the new component

Reply:
created the integration test